connect/resources/fulfillment.py

```diff
@@ -0,0 +1,35 @@
+from connect.config import Config
+from connect.resources.tier_account import TierAccountRequestResource
+from connect.resources.tier_config_request import TierConfigRequestResource
+
+
+class Fulfillment(object):
```

Comment:
There's already a model called Fulfillment. Wouldn't it be a bit confusing having two classes with the same name in two packages?.

Reply:
We are migrating to resources business classes that group functionality according to the portal. 
In this way, was created the classes Fulfillment and Subscription. Could be right rename the model class Fulfillment to AssetRequest.

Reply:
Good. That's the name I gave to the model in the Haxe SDK too.